src/js/compatibility.js

```diff
@@ -0,0 +1,75 @@
+(function(playerLibrary) {
+    // Check if the version of the player requires the compatibility shim
+    if (parseInt(playerLibrary.version, 10) >= 8) {
```

Comment:
This would throw an error if `window.jwplayer` is `undefined`. Though the compatibility script should ideally be added to the page after the jwplayer library, we should be more defensive and check that it's defined.